src/storage.cpp

```diff
@@ -1692,6 +1693,9 @@ namespace libtorrent
 	{
 		TORRENT_ASSERT(m_files.piece_length() > 0);
 
+		if (checked_fastresume) return check_no_fastresume(ec);
```

Comment:
I understand you. Well, I will keep looking, if you have any new idea, let us know.